src/docs/Components/Form/Toggle/index.mdx

```diff
@@ -4,13 +4,15 @@ order: 12
 description: 'A toggle is used to choose between two possibilities and when the user needs instant feedback. It is common to use toggles when you need to show or hide content and "on/off" switch'
 links:
   vue:
-    link: 'https://adeo.github.io/mozaic-vue/?path=/story/components-mtoggle--try-it'
+    link: 'https://adeo.github.io/mozaic-vue/?path=/story/components-mtoggle'
     status: 'ready'
   freemarker:
     status: 'ready'
-    link: 'https://adeo.github.io/integration-web-core--socle/pages/Components/form/switch/'
+    link: 'https://solid-bassoon-011c369c.pages.github.io/pages/3_Components/form/switch/'
+  webComponent:
+    status: 'ready'
+    link: 'https://ubiquitous-giggle-7b5b6e33.pages.github.io/?path=/story/components-toggle'
```

Comment:
The link is broken.
The good url is => https://solid-bassoon-011c369c.pages.github.io/pages/3_Components/form/toggle/